e2e/claims.service.e2e.ts

```diff
@@ -826,7 +827,7 @@ describe('Сlaim tests', () => {
       });
     };
 
-    test('verifyVc should verify a VC with no errors if the issuer is authorized', async () => {
+    test.only('verifyVc should verify a VC with no errors if the issuer is authorized', async () => {
```

Comment:
Probably we don't want the `only` here, right?
```suggestion
    test('verifyVc should verify a VC with no errors if the issuer is authorized', async () => {
```

Reply:
oh, sorry. How do think should we add this https://www.npmjs.com/package/eslint-plugin-no-only-tests?